### ISSUE TITLE
peer.on('open') not triggered on Chrome

### ISSUE THREAD

**Issue by erfanmola** [peers/peerjs#862]
Hi, Here is my code and it used to work !  :

```
let connection = myPeer.connect(TargetPeerID);

connection.on('open', function () {
    console.log("Connection Established");
}
```

And yep, very simple ! it is not logging in console and open is not getting triggered
Im using PeerJS Default server and no extra options, I just obtain ID from PeerJS server, and use this code manually in function to connect, but on('open') is not triggering

Im using latest version of chrome to do this test (OS: Arch Linux)
And the strange thing is this code used to work ! and now its not

Firefox and Opera are Ok, Chrome is having this issue

**Comment by maryamshm** [comment]
Same issue

**Comment by erfanmola** [comment]
The issue seems to be fixed on chrome  91.0.4472.101

**Comment by VitaliiPolushkinDigis** [comment]
same issues for me as now march 2024, use chrome 122 and peerjs latest version
